### PR TITLE
Getting started simplified

### PR DESCRIPTION
**Pull request by ssimon**
Tried to make it easier to get started with hydra, by making use of the debugging package.

### CODE REVIEW

File: stages/debugging/debugging/pom.xml [hunk 1]
```diff
@@ -3,19 +3,24 @@
 	<parent>
 		<groupId>com.findwise.hydra</groupId>
 		<artifactId>stages</artifactId>
-		<version>0.4.0-SNAPSHOT</version>
+		<version>0.3.1</version>
```
Comment (laserval) [inline stages/debugging/debugging/pom.xml]: Is this right?

Comment (ssimon) [inline stages/debugging/debugging/pom.xml]: Hmm.. I guess not.. I think I did that changes for 0.3.1 but it should work
for 0.4.0-SNAPSHOT aswell..

Comment (laserval) [inline stages/debugging/debugging/pom.xml]: Ah, I see, you were working with the 0.3.x branch. Right now there aren't any breaking changes on master I believe, so it should be fine.

But yeah, if it's merging to master and isn't a tagged release, it should be a SNAPSHOT. Btw you set the test-suite artifact to 0.4.0-SNAPSHOT so...

Comment (ssimon) [inline stages/debugging/debugging/pom.xml]: Fixed!

**Pull request merged**